remo/events/migrations/0024_migrate_events_planning_pad.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from django.conf import settings
+
+
+OLD_PAD_URL = 'etherpad.mozilla.org'
+NEW_PAD_URL = 'public.etherpad-mozilla.org'
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """ Migrate the planning pad in events to the new etherpad url."""
+        Event = orm['events.Event']
+        events = Event.objects.filter(planning_pad_url__icontains=settings.ETHERPAD_URL)
+        for event in events:
+            new_planning_pad_url = event.planning_pad_url.replace(OLD_PAD_URL, NEW_PAD_URL)
```

Comment:
In this data migration you are moving everything from ``etherpad.mozilla.org`` to ``public.etherpad-mozilla.org``.

* Some teams (``remo`` for sure) have migrated their pads using this format: ``public.etherpad-mozilla.org/<team>-<name>``.
* What happens with the teams that are not migrated (links will be broken).